src/main/resources/application.properties

```diff
@@ -84,3 +84,6 @@ quarkus.log.cloudwatch.log-stream-name=policies-ui-backend
 quarkus.log.cloudwatch.level=INFO
 quarkus.log.cloudwatch.access-key-id=placeholder
 quarkus.log.cloudwatch.access-key-secret=placeholder
+
+# Environment
+env.name=""
```

Comment:
There's already a default value in the code, this could be removed.

Reply:
I was getting errors without it. I'll retry.

Reply:
Tests fail to start because of this error: 
```
io.quarkus.runtime.configuration.ConfigurationException: Failed to load config value of type class java.lang.String for: env.name
```

Looks like the empty string counts as "nothing is here, fail". 

So according to this discussion: https://github.com/quarkusio/quarkus/issues/2765
Empty value means not set, adding it as Optional allows the value to be empty without failing. 

I'll update the code.